Makefile

```diff
@@ -20,10 +27,17 @@ yarn_pid := $(state_dir)/yarn.pid
 yarn_log := $(state_dir)/yarn.log
 
 js_bindings_dir := daml-ts
+target_dir := target
 
-.PHONE: all
+.PHONY: all package publish
 all: package
 
+publish: package
+	git tag -f "${TAG_NAME}"
+	ghr -replace "${TAG_NAME}" "$(target_dir)/${NAME}.dit"
```

Comment:
What is `ghr` ?

Reply:
Tool for publishing to 'GitHub Repositories'